deploy/crd/nificluster.crd.yaml

```diff
@@ -147,6 +147,7 @@ spec:
                 version:
                   enum:
                     - 1.13.2
+                    - 1.14.0
```

Comment:
I thought we don't support version 1.14 yet.

Reply:
No but fixing that isnt part of this pull request.

Reply:
Yes, but then this PR shouldn't add it to the list of supported versions, right?

Reply:
done